app/views/transferAgreement.scala.html

```diff
@@ -10,61 +10,64 @@
         @progressIndicator(Messages("transferAgreement.progress"))
 
         <h1 class="govuk-heading-xl">@Messages("transferAgreement.title")</h1>
-        @form(action = routes.TransferAgreementController.transferAgreementSubmit(consignmentId)) {
+        @form(
+            routes.TransferAgreementController.transferAgreementSubmit(consignmentId),
+            (Symbol("novalidate"), "")
+        ) {
         @CSRF.formField
         <fieldset class="govuk-fieldset" aria-describedby="transfer-agreement">
-            <div class="govuk-form-group">
-                @inputRadioGroup(
-                transferAgreementForm("publicRecord"),
-                agreementOptions,
-                Symbol("_name") -> Messages("transferAgreement.publicRecord"),
-                Symbol("_requiredOption") -> true,
-                Symbol("_disabledOption") -> "false",
-                Symbol("_selectedOption") -> "true"
-                )
-                @inputRadioGroup(
-                transferAgreementForm("crownCopyright"),
-                agreementOptions,
-                Symbol("_name") -> Messages("transferAgreement.crownCopyright"),
-                Symbol("_requiredOption") -> true,
-                Symbol("_disabledOption ")-> "false"
-                )
-                @inputRadioGroup(
-                transferAgreementForm("english"),
-                agreementOptions,
-                Symbol("_name") -> Messages("transferAgreement.english"),
-                Symbol("_requiredOption") -> true,
-                Symbol("_disabledOption") -> "false"
-                )
-                @inputRadioGroup(
-                transferAgreementForm("digital"),
-                agreementOptions,
-                Symbol("_name") -> Messages("transferAgreement.digital"),
-                Symbol("_requiredOption") -> true,
-                Symbol("_disabledOption") -> "false"
-                )
-            </div>
+            @inputRadioGroup(
+            transferAgreementForm("publicRecord"),
+            agreementOptions,
+            Symbol("_name") -> Messages("transferAgreement.publicRecord"),
+            Symbol("_requiredOption") -> true,
+            Symbol("_checkedOption") -> transferAgreementForm.data.getOrElse("publicRecord", ""),
```

Comment:
Out of curiosity, what are the issues with the Play templates? And how will we remember to fix them?

Reply:
Within the checkboxes partial specifically, we couldn't find a way for it to allow us to use the checked option as is, so we had to convert it to a string. It doesn't feel like that's the best way of doing it, but we couldn't find another way to make it work properly - we found this when doing the bug fix to keep the checkboxes checked even when the form is submitted but not complete.